release_notes/Service_Fabric_ReleaseNotes_10.md

```diff
@@ -0,0 +1,76 @@
+# Microsoft Azure Service Fabric 10.0 Release Notes
+
+* [Service Fabric Packages and Versions](#service-fabric-packages-and-versions)
+* [Service Fabric Feature and Bug Fixes](#service-fabric-feature-and-bug-fixes)
+* [Retirement and Deprecation Path Callouts](#retirement-and-deprecation-path-callouts)
+* [Repositories and Download Links](#repositories-and-download-links)
+
+## Service Fabric Packages and Versions
+The following packages and versions are part of this release:
+
+| Service | Platform | Version |
+|---------|----------|---------|
+|Service Fabric Runtime| Ubuntu 18 <br> Ubuntu 20 <br> Windows | 10.0.1728.1 <br> 10.0.1728.1 <br> 10.0.1816.9590 |
+|Service Fabric for Windows Server|Service Fabric Standalone Installer Package | 10.0.1816.9590 |
+|.NET SDK |Windows .NET SDK <br> Microsoft.ServiceFabric <br> Reliable Services and Reliable Actors <br> ASP.NET Core Service Fabric integration| 7.0.1816  <br> 10.0.1816 <br> 7.0.1816 <br> 7.0.1816 |
+|Java SDK  |Java for Linux SDK  | 1.0.6 |
+|Service Fabric PowerShell and CLI | AzureRM PowerShell Module  <br> SFCTL |  0.3.15  <br> 11.0.1 |
+
+
```

Comment:
Also add a section here for the Breaking Changes. Bring the 4 breaking changes in the table below to up here as a separate table with those 4 items.